src/common/updating/toast_notifications_helper.cpp

```diff
@@ -0,0 +1,127 @@
+#include "pch.h"
+
+#include "toast_notifications_helper.h"
+
+#include <common/notifications.h>
+
+#include "updating.h"
+
+#include "VersionHelper.h"
+#include "version.h"
+
+namespace
+{
+    const wchar_t UPDATE_NOTIFY_TOAST_TAG[] = L"PTUpdateNotifyTag";
+    const wchar_t UPDATE_READY_TOAST_TAG[] = L"PTUpdateReadyTag";
+}
+
+namespace localized_strings
+{
+    const wchar_t GITHUB_NEW_VERSION_AVAILABLE[] = L"An update to PowerToys is available.\n";
+    const wchar_t GITHUB_NEW_VERSION_DOWNLOAD_STARTED[] = L"PowerToys download started.\n";
+    const wchar_t GITHUB_NEW_VERSION_READY_TO_INSTALL[] = L"An update to PowerToys is ready to install.\n";
+    const wchar_t GITHUB_NEW_VERSION_DOWNLOAD_INSTALL_ERROR[] = L"Error: couldn't download PowerToys installer. Visit our GitHub page to update.\n";
+    const wchar_t GITHUB_NEW_VERSION_UPDATE_NOW[] = L"Update now";
+    const wchar_t GITHUB_NEW_VERSION_UPDATE_AFTER_RESTART[] = L"At next launch";
+    
+    const wchar_t UNINSTALLATION_SUCCESS[] = L"Previous version of PowerToys was uninstalled successfully.";
+    const wchar_t UNINSTALLATION_UNKNOWN_ERROR[] = L"Error: please uninstall the previous version of PowerToys manually.";
+
+    const wchar_t GITHUB_NEW_VERSION_AVAILABLE_OFFER_VISIT[] = L"An update to PowerToys is available. Visit our GitHub page to update.\n";
+    const wchar_t GITHUB_NEW_VERSION_UNAVAILABLE[] = L"PowerToys is up to date.\n";
+    const wchar_t GITHUB_NEW_VERSION_VISIT[] = L"Visit";
+    const wchar_t GITHUB_NEW_VERSION_MORE_INFO[] = L"More info...";

```

Comment:
Please remove "..." for an consistent look and feel.

Reply:
Maybe "information" is a better spelling than "info"?

Reply:
"More info..." looks good, we should better leave it unchanged.